test/unit/math/prim/scal/fun/init_threadpool_tbb_test.cpp

```diff
@@ -0,0 +1,26 @@
+#include <stan/math/prim/core.hpp>
+
+#include <gtest/gtest.h>
+#include <test/unit/util.hpp>
+#include <test/unit/math/prim/mat/functor/utils_threads.hpp>
+
+#include <tbb/task_scheduler_init.h>
+#include <tbb/task_arena.h>
```

Comment:
These two should be inside `#ifdef STAN_THREADS`

Reply:
thx for catching that.